File: apps/flights-service-admin/src/api/airline/AirlineCreateInput.ts
import { FlightWhereUniqueInput } from "../flight/FlightWhereUniqueInput";
import { FlightCreateNestedManyWithoutAirlinesInput } from "./FlightCreateNestedManyWithoutAirlinesInput";

export type AirlineCreateInput = {
  arrivalFlights?: FlightWhereUniqueInput | null;
  country?: string | null;
  flights?: FlightCreateNestedManyWithoutAirlinesInput;
  name?: string | null;
};

File: apps/flight-catalog/src/APIs/Airline/Dtos/AirlineCreateInput.cs
namespace FlightCatalog.APIs.Dtos;

public class AirlineCreateInput
{
    public string? Country { get; set; }

    public DateTime CreatedAt { get; set; }

    public List<Flight>? Flights { get; set; }

    public string? Id { get; set; }

    public string? Name { get; set; }

    public DateTime UpdatedAt { get; set; }
}
